PReLU.cu

```diff
@@ -0,0 +1,263 @@
+#include "THCApply.cuh"
+#include "THCReduce.cuh"
+#include "utils.h"
+
+// This is bad, because the following is defined in SpatialConvolution.cu ...
```

Comment:
I agree we should compile each file separately. We do it in cutorch now, which enables us to compile files in parallel.